app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -174,7 +174,7 @@ def check_policy_prevent(policy_event, event_target, cb_method)
       :class_name  => self.class.to_s,
       :instance_id => id,
       :method_name => :check_policy_prevent_callback,
-      :args        => [cb_method, event_target],
+      :args        => [cb_method, event_target.class.name, event_target.id],
```

Comment:
Looking at other `check_policy_prevent` functions (host, vm_or_template), they don't take target, just `*cb_method` and use `[*cb_method]` here.
You could the same and then `scan_job_create` would have to do
```
     check_policy_prevent(:request_containerimage_scan, :raw_scan_job_create, entity.class.name, entity.id)
```
I like that better because one can understand `scan_job_create` -> `raw_scan_job_create` path without looking into the policy methods.
Plus less obstacles to dedup these functions one day...  (AFAICT the main diffrence is in building inputs hashes for `MiqEvent.raise_evm_event`.)